test/integration/index.spec.js

```diff
@@ -185,4 +186,22 @@ describe('[INTEGRATION] index', function () {
       createCompiler({ tslint: true });
     }).to.not.throw.error;
   });
+
+  it('should not find syntactic errors when checkSyntacticErrors is false', function (callback) {
+    var compiler = createCompiler({ checkSyntacticErrors: false }, true);
+    
+    compiler.run(function(error, stats) {
+      expect(stats.compilation.errors.length).to.be.equal(2);
+      callback();
+    });
+  });
+
+  it('should not find syntactic errors when checkSyntacticErrors is true', function (callback) {
```

Comment:
It should be `should find` not `should not find` :)

Reply:
quite right!